.vuepress/components/Home.vue

```diff
@@ -169,6 +170,26 @@ a
 
     }
 
+    &__tags {
+      flex-direction: row;
+
+      &__item {
+        display: inline-block;
+        margin-right: 0.5rem;
+        color: var(--color-text-dim, inherit);
+        border: 1px solid rgba(0,0,0,0.07);
+        font-size: 0.875rem;
+        line-height: 1.25rem;
+        letter-spacing: 0.03em;
+        text-transform: capitalize;
+        background: var(--tag-background-color);
+        border-radius: 0.25rem;
+        padding: 0.25rem 0.5rem 0.125rem;
+        width: fit-content;
+      }
+
```

Comment:
```suggestion
```

Reply:
(just to trigger a rebuild to see if marko's changes to CI fix this or caused this)